public/index.html

```diff
@@ -27,7 +27,7 @@
         crossorigin="anonymous">
 
     <!-- Custom styles -->
-    <link rel="stylesheet" type="text/css" href="/styles/style.css">
+    <link rel="stylesheet" type="text/css" href="/static/styles/style.css">
```

Comment:
Just checking: is `/static` correct here? Since there are no actual directories with that name... I assume there is some rewrite going on.